processor/otel/consumer.go

```diff
@@ -633,6 +671,14 @@ func translateSpan(span pdata.Span, metadata model.Metadata, event *model.Span)
 		event.DB = &db
 	case isMessagingSpan:
 		event.Type = "messaging"
+		event.Subtype = message.System
+		event.Action = message.Operation
+		event.Name = message.Name()
```

Comment:
We use the OpenTelemetry span's name everywhere else - I think we should do the same here (and not set `event.Name` here.)